src/Valit/Rules/ValitRules.cs

```diff
@@ -1,26 +1,26 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
+using Valit.Strategies;
 
 namespace Valit
 {
     public class ValitRules<TObject> : IValitRules<TObject>, IValitRulesStrategyPicker<TObject> where TObject : class
     {
         private TObject _object;       
         private readonly List<IValitRule<TObject>> _rules;
-        private ValitRulesStrategies _strategy;
+        private IValitStrategy _strategy;
 
         private ValitRules(IEnumerable<IValitRule<TObject>> rules)
         {   
             _rules = rules?.ToList() ?? new List<IValitRule<TObject>>();
-            _strategy = ValitRulesStrategies.Complete;
+            _strategy = default(DefaultValitStrategies).Complete;
```

Comment:
I guess I should have used new here in case someone later on decides to turn my struct into a class for unknown reason.